scss/_navbar.scss

```diff
@@ -174,6 +174,20 @@
   }
 }
 
+// Navigation items into toggleable navbar
+.navbar-nav {
+  .nav-item {
```

Comment:
We're generally trying to avoid nesting, especially more than 1 level of it.
Is the `.navbar-nav .nav-item` part really necessary?